pkg/model/azuremodel/api_loadbalancer.go

```diff
@@ -17,9 +17,16 @@ limitations under the License.
 package azuremodel
 
 import (
-	"errors"
+	"fmt"
+	"sort"
 
+	"github.com/Azure/go-autorest/autorest/to"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog"
```

Comment:
Ah, we need to change this to v2.. (`"k8s.io/klog/v2"`)..

Reply:
just updated that in that file and the loadbalancer.go file in azuretasks. let me know if you see any that i might have missed.